### PR TITLE
Easter egg

### PR DESCRIPTION
**Pull request by gilmanov-ildar**
When a user clicks 5 times in 1.5 seconds to the Menu banner we switch 
developer mode flag and show message box to user.

image

image

### CODE REVIEW

**Comment by gilmanov-ildar** [comment]
Related to https://github.com/MasterTailer/DIGITAiL/issues/5

File: src/kirigami-icons.qrc [hunk 1]
```diff
@@ -24,5 +24,6 @@
         <file alias="icons/list-remove.svg">../3rdparty/breeze-icons/icons/actions/22/list-remove.svg</file>
         <file alias="icons/document-edit.svg">../3rdparty/breeze-icons/icons/actions/22/document-edit.svg</file>
         <file alias="icons/accept_time_event.svg">../3rdparty/breeze-icons/icons/actions/22/accept_time_event.svg</file>
+        <file alias="icons/gnumeric-formulaguru.svg">../3rdparty/breeze-icons/icons/actions/16/gnumeric-formulaguru.svg</file>
```
Comment (leinir) [inline src/kirigami-icons.qrc]: i'd quite like to not be using any random picture for things... but we can swap those out later anyway

Comment (gilmanov-ildar) [reply src/kirigami-icons.qrc]: It is not a random picture, I reviewed available images and thought that this one is more suitable for Developer mode than other, but off course we can change it later :)

Comment (leinir) [reply src/kirigami-icons.qrc]: Sure, i didn't mean to say you picked an icon that was completely without reason :D What i meant is there are icons in Breeze specifically designed to suggest "development" and the like. Not sure what you're using, but just to make sure you're aware of that one, Cuttlefish is a handy little application for looking through icon sets and such, allowing you to search by name and such, for example for "develop" or "code", yielding things like this screenshot. But, at any rate, all merged and whatnot, we can work on that later, not going to be painting any bike sheds today ;)
image

Comment (gilmanov-ildar) [reply src/kirigami-icons.qrc]: Yup, the `{}` is much more suitable :)

**Pull request merged**